### DISCUSSION TITLE
Apple Codesign 0.17.0

### DISCUSSION THREAD

**Discussion by indygreg** [indygreg/PyOxidizer#621] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/indygreg/PyOxidizer/releases/tag/apple-codesign/0.17.0'>Apple Codesign 0.17.0</a>.</em>

**Comment by dawmster** [comment]
Hi, can you use your tool for signing/preparing iOS app?
I know it still needs go through App Store Connect, and be reviewed, but still it would be very useful for apps made outside XCode - in Python for example.
